topics:
  input:
    GT_pose: "/qualisys/Anafi/pose"
    anafi_rpy: "anafi/rpy"
    anafi_pose: "anafi/pose"


